static/app/components/events/contexts/runtime/getRuntimeKnownData.tsx

```diff
@@ -1,33 +1,41 @@
-import {getMeta} from 'sentry/components/events/meta/metaProxy';
-import {KeyValueListData} from 'sentry/types';
+import {Event, KeyValueListData} from 'sentry/types';
 import {defined} from 'sentry/utils';
 
-import getRuntimeKnownDataDetails from './getRuntimeKnownDataDetails';
-import {RuntimeData, RuntimeKnownDataType} from './types';
+import {getRuntimeKnownDataDetails} from './getRuntimeKnownDataDetails';
+import {runtimeKnownDataValues} from './index';
+import {RuntimeData} from './types';
 
-function getRuntimeKnownData(
-  data: RuntimeData,
-  runTimerKnownDataValues: Array<RuntimeKnownDataType>
-): KeyValueListData {
+type Props = {
+  data: RuntimeData;
+  meta: NonNullable<Event['_meta']>['user'];
+};
+
+export function getRuntimeKnownData({data, meta}: Props): KeyValueListData {
   const knownData: KeyValueListData = [];
 
-  const dataKeys = runTimerKnownDataValues.filter(runTimerKnownDataValue =>
-    defined(data[runTimerKnownDataValue])
-  );
+  const dataKeys = runtimeKnownDataValues.filter(runTimerKnownDataValue => {
```

Comment:
```suggestion
  const dataTypes = runtimeKnownDataValues.filter(runTimerKnownDataValue => {
```
wdyt?

Reply:
most of the code has dataKeys as a const... do you mind if I address this in a separate PR? Then I can update all the occurrences

Reply:
yeah sounds good.